services/QuillLMS/client/app/bundles/Teacher/containers/UploadRosters.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+import XLSX from 'xlsx'
+import request from 'request'
+
+import getAuthToken from '../components/modules/get_auth_token'
+
+
+export const UploadRosters = () => {
+
+  const [schoolId, setSchoolId] = React.useState<Number>();
+  const [teachers, setTeachers] = React.useState<Array<any>>([]);
+  const [students, setStudents] = React.useState<Array<any>>([])
+
+  function handleSchoolIdChange(e) {
+    setSchoolId(e.target.value)
+  }
+
+  function handleChangeFile(file) {
+    const fileReader = new FileReader();
+    fileReader.onload = (e) => {
+      const data = new Uint8Array(e.target.result);
+      const workbook = XLSX.read(data, { type: 'array', });
+
+      const sheet1 = workbook.Sheets[workbook.SheetNames[0]]
+      const sheet1_array = XLSX.utils.sheet_to_json(sheet1, {header:1})
```

Comment:
This should probably be camelCase because it's in a Javascript file! Same for the `sheet2_array` variable.